### ISSUE TITLE
What do I put for US-West region?

### ISSUE THREAD

**Issue by utgarda** [solutious/rudy#52]
Putting default zones 'us-west-1a' , 'us-west-1' and 'us-west' I get the following:
    > rudy -vv startup
    Rudy::AWS::EC2::InvalidRegion: us-west-1
    > rudy -vv startup
    Rudy::AWS::EC2::InvalidRegion: us-west-
    > rudy -vv startup
    Rudy::AWS::EC2::InvalidRegion: us-wes

**Comment by delano** [comment]
That's odd. Try explicitly setting the region to 'us-west-1'.

```
defaults do
  region 'us-west-1'
  zone 'us-west-1a'
end
```

**Comment by utgarda** [comment]
Same thing:
    > rudy -vvv --debug shutdown
    Loading config from /home/etsvigun/.rudy/config
    Loading config from /home/etsvigun/devenv/biotech/biotech/erlang/poc/Rudyfile
    Rudy::AWS::EC2::InvalidRegion: us-west-1
    /usr/lib64/ruby/gems/1.8/gems/rudy-0.9.8.016/bin/../lib/rudy/aws/ec2.rb:11:in `connect'

Defaults here below, no more mentioning region in that file but this:

```
defaults do
```

# zone :'us-east-1a'

```
  region 'us-west-1'
  zone 'us-west-1a'
  environment :stage
  role :master
  color true                         # Terminal colors? true/false
end
```

**Comment by delano** [comment]
Ah! My bad. The problem is here:

https://github.com/solutious/rudy/blob/master/lib/rudy/aws.rb#L9

I'll fix it now.

**Comment by delano** [comment]
Fixed in 0.9.8.017.
